godot-core/src/bind.rs

```diff
@@ -24,6 +24,66 @@ use crate::obj::GodotClass;
 ///
 /// Do not call any of these methods directly -- they are an interface to Godot. Functionality
 /// described here is available through other means (e.g. `init` via `Gd::new_default`).
+///
+/// It is not enough to impl GodotExt to be registered in Godot, for this you should look at [ExtensionLibrary](crate::init::ExtensionLibrary)
+///
+/// # Examples
+///
+/// ## Basic Reference
```

Comment:
It's called `RefCounted` now (`Reference` in Godot 3).

And what do you mean with "basic"? Maybe be explicit and say:
```rs
/// ## With `RefCounted` as a base class
```